client/my-sites/people/team-list/team.jsx

```diff
@@ -5,76 +5,87 @@
  */
 import deterministicStringify from 'fast-json-stable-stringify';
 import { localize } from 'i18n-calypso';
-import { omit } from 'lodash';
 import React from 'react';
-import debugFactory from 'debug';
 import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import { Card } from '@automattic/components';
 import PeopleListItem from 'calypso/my-sites/people/people-list-item';
-import { fetchUsers } from 'calypso/lib/users/actions';
 import InfiniteList from 'calypso/components/infinite-list';
 import NoResults from 'calypso/my-sites/no-results';
 import PeopleListSectionHeader from 'calypso/my-sites/people/people-list-section-header';
 import ListEnd from 'calypso/components/list-end';
 import { recordGoogleEvent } from 'calypso/state/analytics/actions';
 
-const debug = debugFactory( 'calypso:my-sites:people:team-list' );
-
 class Team extends React.Component {
 	infiniteList = React.createRef();
 
-	isLastPage = () =>
-		this.props.totalUsers <= this.props.users.length + this.props.excludedUsers.length;
+	renderPerson = ( user ) => (
+		<PeopleListItem key={ user.ID } user={ user } type="user" site={ this.props.site } />
+	);
+
+	getPersonRef = ( user ) => 'user-' + user.ID;
+
+	renderLoadingPeople = () => <PeopleListItem key="people-list-item-placeholder" />;
 
 	render() {
-		const key = deterministicStringify( omit( this.props.fetchOptions, [ 'number', 'offset' ] ) );
+		const {
+			site,
+			users,
+			fetchOptions,
+			search,
+			fetchingUsers,
+			fetchingNextPage,
+			totalUsers,
+			hasNextPage,
+			fetchNextPage,
+			translate,
+		} = this.props;
+
+		const { number, offset, ...keyProps } = fetchOptions;
+		const key = deterministicStringify( keyProps );
```

Comment:
I would extract this key up to the parent component. Avoids passing an ambiguous `fetchOptions` prop, and lets us replace it with the more specific `search` and `listKey` props, the only parts we really use.

Reply:
Done via e51755f